### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by haroldmud**
## In this branch I did the following:  :point_down: 

- Created a database named `vet_clinic`.
- Created a table named animals with the given columns.  
- Inserted the given data.
- Wrote queries for the following quests :
  - Found all animals whose name ends in "mon".
  $mon
  - Listed the name of all animals born between 2016 and 2019. 
$birthname
  - Listed the name of all animals that are neutered and have less than 3 escape attempts.
$less3
  - Listed the date of birth of all animals named either "Agumon" or "Pikachu".
$birthdate

  - Listed name and escape attempts of animals that weigh more than 10.5kg
$namescape
  - Found all animals that are neutered.
   $neutral
  - Found all animals not named Gabumon.
$nogabumon
  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
  $weight

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Pikachu'; -----> this is a request
+SELECT * FROM animals; ---> to check whrther the updated item changed or not
+
+SELECT * FROM animals WHERE Name like '%mon';
+
+SELECT * FROM animals WHERE Date_of_birth >= '2016-02-25' AND Date_of_birth <= '2019-02-27';
```
Comment (keneogu) [inline queries.sql]: It should list all animals born between 2016 and 2019.
`2016-01-01` to `2019-12-31`

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+SELECT * FROM animals WHERE Escape_attempts > 3 AND Neutered = 'true';
+
+SELECT * FROM animals WHERE Name = 'Agumon' OR Name = 'Pikachu';
```
Comment (keneogu) [inline queries.sql]: it should list Date of birth of all animals named either "Agumon" or "Pikachu".

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @haroldmud,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 3]
```diff
@@ -4,11 +4,11 @@ SELECT * FROM animals; ---> to check whrther the updated item changed or not
 
 SELECT * FROM animals WHERE Name like '%mon';
 
-SELECT * FROM animals WHERE Date_of_birth >= '2016-02-25' AND Date_of_birth <= '2019-02-27';
+SELECT * FROM animals WHERE Date_of_birth >= '2016-01-01' AND Date_of_birth <= '2019-12-31';
```
Comment (Cathella) [inline queries.sql]: [REQUIRED] Kindly return the names of animals as mentioned in the requirements. Please refer to the following query

```ruby
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @haroldmud 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+UPDATE animals SET Neutered = 'false' WHERE Name = 'Pikachu'; -----> this is a request
+SELECT * FROM animals; ---> to check whrther the updated item changed or not
+
+SELECT * FROM animals WHERE Name like '%mon';
+
+SELECT * FROM animals WHERE Date_of_birth >= '2016-01-01' AND Date_of_birth <= '2019-12-31';
```
Comment (codepantha) [inline queries.sql]: - [ ] Please make the change suggested by the previous reviewer about returning the names of animals. Please refer to the following query 👇 
```
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @haroldmud 👋 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



### Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Correct GitFlow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @haroldmud  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviews

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**